elements/nuxeo-collections/nuxeo-favorites.html

```diff
@@ -128,9 +128,7 @@
           <div tabindex$="{{tabIndex}}" class$="[[_computedClass(selected)]]">
             <div class="list-item-box">
               <div class="list-item-info">
-                <div class="list-item-thumb">
```

Comment:
This seems to be a typo, since what we have an unused `list-item-thumbnail` class. I'd say this was the original intention:
```suggestion
                <div class="list-item-thumbnail">
```